libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -196,6 +196,14 @@ const AP_Param::GroupInfo AP_MotorsHeli_Dual::var_info[] = {
     // @Increment: 1
     AP_SUBGROUPINFO(_swashplate2, "SW2_", 21, AP_MotorsHeli_Dual, AP_MotorsHeli_Swash),
 
+    // @Param: DCP_TRIM
+    // @DisplayName: Differential Collective Pitch Trim
+    // @Description: Removes I term bias due to center of gravity offsets or discrepancies between rotors in swashplate setup.
```

Comment:
I wonder if a better description might be to describe the physical difference it makes first.  For example negative numbers lower the 2nd swash vs the 1st.

Reply:
@rmackay9  I considered that but thought it might get too wordy.  Would it make sense to say as a second sentence.  “if DCP axis has I term bias while hovering in calm winds, use value of bias in DCP_TRIM to re-center I term.”

Reply:
ok, sounds good to me!